unittest/action_factory.hpp

```diff
@@ -65,7 +65,7 @@ class ActionModelFactory {
 
   ~ActionModelFactory() {}
 
-  boost::shared_ptr<crocoddyl::ActionModelAbstract> get_action_model() { return action_model_; }
+  boost::shared_ptr<crocoddyl::ActionModelAbstract> get_action() { return action_; }
```

Comment:
I believe this naming is bad. I suggest to rename these getter-factory functions (i.e. for all factory) to create().